libplatsupport/plat_include/pc99/platsupport/plat/serial.h

```diff
@@ -6,6 +6,9 @@
 
 #pragma once
 
+#include <autoconf.h>
```

Comment:
Is this related to the fix about with the broken numbers? If this just adds missing includes (as a nice-to-have, that kicks in in certain cases?), consider making this separate commit.

Reply:
@abrandnewusername could you comment on this?